pkg/roachprod/vm/vm.go

```diff
@@ -62,6 +62,12 @@ const (
 	DisksInitializedFile = "/mnt/data1/" + InitializedFile
 )
 
+// UnimplementedError is returned when a method is not implemented by a
+// provider. An error is returned instead of panicking to isolate failures to a
+// single test (in the context of `roachtest`), otherwise the entire test run
+// would fail.
+var UnimplementedError = errors.New("unimplemented")
```

Comment:
I'm considering if this should maybe rather be `rperrors.TransientFailure(errors.New("unimplemented"), "provider_problem")`, or maybe a different class of errors to route it differently.

Reply:
I wouldn't be opposed to that. I think if these errors did happen, there's a decent chance it would go unnoticed in the daily batch of cluster creation errors. But this error in particular should always be fixed.